{{cookiecutter.project_name}}/pipeline.yaml

```diff
@@ -21,28 +21,27 @@ Parameters:
       Type: AWS::SSM::Parameter::Value<String>
       Default: /service/{{cookiecutter.project_name.lower().replace(' ', '-')}}/github/repo
 
-    GithubToken:
-      Description: Github OAuth Token with full permissions on admin:repo_hook and repo
+    GithubTokenSecretArnSuffix:
```

Comment:
Can you give an example in the description? sorry but it doesn't sound clear to me yet.